star_ratings/apps.py

```diff
@@ -13,3 +13,6 @@ def ready(self):
 
         post_save.connect(calculate_ratings, sender=UserRating)
         post_delete.connect(calculate_ratings, sender=UserRating)
+
+        # Add System checks
+        from .checks import rerate_check  # NOQA
```

Comment:
Personally I would prefer calling register in here rather than having a decorator to avoid the side effect on importing similar to how we handle the signals above

Reply:
👍 makes sense.

Reply:
FYI, I've updated and am resolving conflicts then will merge down